addons/l10n_it_edi/tests/test_res_partner.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.exceptions import UserError
+from odoo.tests.common import TransactionCase, tagged
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestResPartner(TransactionCase):
+
+    def test_validate_fiscal_code(self):
+        valid_codes = [
+            "AORTHV05P30V295L",
+            "SPDTHB43S93F42VH",
+            "MDRTUV99H14X2MNU",
+            "XPTDRX73R64YPLUD",
+            "LOLXDR40T3MZRTSV",
+            "GJTIUG55DLQZRTSS",
+            "CDEOTG5PBLQZRTSE",
+            "PERTLELPALQZRTSN",
+            "IT12345678887",
```

Comment:
We should not support ITXXXXXXX as codice

Reply:
Why that?

Reply:
because IT123456789 and 123456789 would be equivalent.  (there is plenty of code depending on identifying with the codice fiscale correctly) @guva-odoo

Reply:
Ok, I add a check for that too